config/default.yml

```diff
@@ -3148,8 +3152,8 @@ Style/NestedParenthesizedCalls:
                  argument list of another parenthesized method call.
   Enabled: true
   VersionAdded: '0.36'
-  VersionChanged: '0.50'
-  Whitelist:
+  VersionChanged: '0.77'
+  Allowlist:
```

Comment:
For this and the remaining options - something like `AllowedMethods` sounds better.